### PR TITLE
feat(cli): added -v support in cli to get version 

### LINKED ISSUE #6763: Add support for -v in the wing-cli
### Use Case

Very minor issue, but might be nice to support `-v` on the wing-cli to get the version of wing. 

Wing currently supports `wing -V`, be nice to support `wing -v` too.

### Proposed Solution

Desired experience would be to support both `-V` and `-v` to get the installed version of wing-cli.

### Implementation Notes

Similar to how we have it implemented for `-V`, just extend for `-v`

### Component

CLI

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.
* If this issue is labeled **needs-discussion**, it means the spec has not been finalized yet. Please reach out on the #dev channel in the [Wing Discord](https://t.winglang.io/discord).

### PR DESCRIPTION
**Pull request by boyney123**
fixes #6763

Wing CLI currently supports `-V` and `--version`.

This adds the ability to `-v` to get the current version.

## Checklist

- [ ] Title matches [Winglang's style guide](https://www.winglang.io/contributing/start-here/pull_requests#how-are-pull-request-titles-formatted)
- [ ] Description explains motivation and solution
- [ ] Tests added (always)
- [ ] Docs updated (only required for features)
- [ ] Added `pr/e2e-full` label if this feature requires end-to-end testing

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

**Review by tsuf239 — APPROVED** [review]

File: apps/wing/src/cli.ts [hunk 1]
```diff
@@ -93,7 +93,7 @@ async function main() {
     showGlobalOptions: true,
   });
 
-  program.name("wing").version(PACKAGE_VERSION);
+  program.name("wing").version(PACKAGE_VERSION, "-v, --version, -V", "output the current version");
```
Comment (MarkMcCulloh) [inline apps/wing/src/cli.ts:96]: I think commander expects all the short flags first

```ts
program.name("wing").version(PACKAGE_VERSION, "-v, -V, --version", "output the current version");
```

Comment (boyney123) [reply apps/wing/src/cli.ts:96]: Thanks, just tested a combination of them, and looks like commander only honours the first 2 in this list (for some reason)...

Comment (MarkMcCulloh) [reply apps/wing/src/cli.ts:96]: Ah, I see. To be honest I would get rid of `-V` for now and just update the docs to `-v`. `-V` for version is pretty atypical imo

Comment (boyney123) [reply apps/wing/src/cli.ts:96]: Cool PR open, thanks https://github.com/winglang/wing/pull/6841

**Pull request merged**